components/kyma-environment-broker/common/hyperscaler/shared_pool.go

```diff
@@ -31,7 +30,7 @@ func (sp *sharedAccountPool) SharedCredentialsSecretBinding(hyperscalerType Type
 	labelSelector := fmt.Sprintf("shared=true,hyperscalerType=%s", hyperscalerType)
 	secretBindings, err := sp.getSecretBindings(labelSelector)
 	if err != nil {
-		return nil, errors.Wrap(err, "getting secret binding")
+		return nil, fmt.Errorf(" getting secret binding: %w", err)
```

Comment:
nit: is the space intentional?
```suggestion
		return nil, fmt.Errorf("getting secret binding: %w", err)
```